src/worker/SandboxedWorker.js

```diff
@@ -49,7 +49,15 @@ class SandboxedWorker extends EventTarget {
     if (!workerFrameSandboxDisabled) {
       this.iframe.sandbox = workerFrameSandbox
     }
-    this.iframe.style = 'position: absolute; width: 0; height: 0; opacity:0;'
+    this.iframe.style = `
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 0;
+      height: 0;
+      opacity: 0;
+      border: 0;
```

Comment:
Was it showing? Odd that with `opacity: 0` and no dimensions this would render, but good catch!

Reply:
Not showing itself, but it was pushing the app 4px up (due to recent changes to the layout).

Reply:
Ahh, makes sense, then very good catch.